simplyblock_web/blueprints/snode_ops_k8s.py

```diff
@@ -555,3 +555,21 @@ def apply_config():
         core_utils.set_hugepages_if_needed(numa, num_pages)
 
     return utils.get_response(True)
+
+@api.post('/bind_device_to_spdk')
```

Comment:
This is the same function as in `snode_ops` if I'm not mistaken. Can we either import it and use it here:
```
from . import snode_ops
...

@api.post('/bind_device_to_spdk')(snode_ops.bind_device_to_spdk)
```
or delegate to a helper?

Reply:
I agree, much better to import rather than to duplicate code